examples/layer-browser/src/examples/core-layers.js

```diff
@@ -384,6 +404,7 @@ export default {
     'GeoJsonLayer (Extruded)': GeoJsonLayerExtrudedExample,
     PolygonLayer: PolygonLayerExample,
     PathLayer: PathLayerExample,
+    'PathLayer (Flat)': PathLayerBinaryExample,
```

Comment:
Side note: We need some data dropdown in the layer browser so we don't get too many copies of each layer in the side bar.